### ISSUE TITLE
listener interface: report total number of tests before starting

### ISSUE THREAD

**Issue by yanne** [robotframework/RIDE#272]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=237) by bryan.oakley on 15 Sep 2009*

If possible, it would be nice for a listener to be given the total number
of tests to be run before the first test starts. With this information a UI
could display a useful progress bar (eg: test X of Y in suite A of B). 

Perhaps this information could be sent when calling the start_suite method
of the listener. Though, if more than one suite is to be run, also knowing
how many suites are to be run would be good, too.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=237#c1) by @pekkaklarck on 15 Sep 2009*


This is a wrong tracker for RF issues. Additionally this is also a duplicate of
<a href="https://code.google.com/p/robotframework/issues/detail?id=321" rel="nofollow">https://code.google.com/p/robotframework/issues/detail?id=321</a>
